stac_geoparquet/arrow/_delta_lake.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import itertools
+from pathlib import Path
+from typing import TYPE_CHECKING, Iterable, Optional, Union
+
+import pyarrow as pa
+from deltalake import write_deltalake
+
+from stac_geoparquet.arrow._api import parse_stac_ndjson_to_arrow
+from stac_geoparquet.arrow._to_parquet import create_geoparquet_metadata
+
+if TYPE_CHECKING:
+    from deltalake import DeltaTable
+
+
+def parse_stac_ndjson_to_delta_lake(
```

Comment:
should we make a generic arrow_batches_to_delta_lake that takes in batches and then just make the parse_stac_ndjson_to_delta_lake a wrapper around that?

Reply:
well, that generic `arrow_batches_to_delta_lake` function is literally just `write_deltalake`. You can pass an `Iterable[pa.RecordBatch]` directly to it. (You just also need to know the schema separately. Are you suggesting a helper that takes the first batch and passes its schema to `write_deltalake`?)